articles/openshift/openshift-faq.md

```diff
@@ -10,196 +10,220 @@ ms.date: 11/04/2019
 
-Advanced/Custom scheduling is currently unsupported. Refer to the [Scheduling documentation](https://docs.openshift.com/container-platform/3.11/admin_guide/scheduling/index.html) for more details.
+- [Exposing a registry](https://docs.openshift.com/aro/4/registry/securing-exposing-registry.html)
+- [Accessing the registry](https://docs.openshift.com/aro/4/registry/accessing-the-registry.html)
 
-## If we scale up the deployment, how do Azure fault domains map into pod placement to ensure all pods for a service do not get knocked out by a failure in a single fault domain?
+For 3.11 clusters the Docker image registry is available. The Docker registry is available from `https://docker-registry.apps.<clustername>.<region>.azmosa.io/`. You can also use Azure Container Registry.
 
-There are by default five fault domains when using virtual machine scale sets in Azure. Each virtual machine instance in a scale set will get placed into one of these fault domains. This ensures that applications deployed to the compute nodes in a cluster will get placed in separate fault domains.
+## Networking
 
-Refer to [Choosing the right number of fault domains for virtual machine scale set](https://docs.microsoft.com//azure/virtual-machine-scale-sets/virtual-machine-scale-sets-manage-fault-domains) for more details.
+### Can I deploy a cluster into an existing virtual network?
 
-## Is there a way to manage pod placement?
+In 4.x clusters you can deploy a cluster into an existing VNet.
 
-Customers have the ability to get nodes and view labels as the customer-admin.  This will provide a way to target any VM in the scale set.
+In 3.11 clusters you cannot deploy a cluster into an existing VNet. You can connect an Azure Red Hat OpenShift 3.11 cluster to an existing VNet via peering.
 
-Caution must be used when using specific labels:
+### Is cross-namespace networking supported?
+
+Customer and individual project admins can customize cross-namespace networking (including denying it) on a per project basis using `NetworkPolicy` objects.
+
+### I am trying to peer into a virtual network in a different subscription but getting Failed to get VNet CIDR error.
+
+In the subscription that has the virtual network, make sure to register `Microsoft.ContainerService` provider with the following command: `az provider register -n Microsoft.ContainerService --wait`
+
+### Can we specify IP ranges for deployment on the private VNet, avoiding clashes with other corporate VNets once peered?
+
+In 4.x clusters you can specify your own IP ranges.
+
+In 3.11 clusters Azure Red Hat OpenShift supports VNet peering and allows the customer to provide a VNet to peer with and a VNet CIDR in which the OpenShift network will operate.
+
+The VNet created by ARO will be protected and will not accept configuration changes. The VNet that is peered is controlled by the customer and resides in their subscription.
```

Comment:
expand VNet to virtual network

Reply:
I expanded ARO. Can we double check on expanding VNet? I did a quick search of the source and I see this used pretty extensively (11 times in the aks section).

Happy to do it if that's the call, but it feels a bit cumbersome to the reader for this specific term.